etl/steps/data/garden/wb/2022-10-29/wb_gender/wb_gender.countries.json

```diff
@@ -219,7 +218,16 @@
   "Zambia": "Zambia",
   "Zimbabwe": "Zimbabwe",
   "High income": "High-income countries",
-  "Lower middle income": "Lower-middle-middle income countries",
+  "Lower middle income": "Lower-middle-income countries",
   "Low income": "Low-income countries",
-  "Upper middle income": "Upper-middle-income countries"
+  "Upper middle income": "Upper-middle-income countries",
+  "North America": "North America (WB)",
+  "Latin America & Caribbean": "Latin America and Caribbean (WB)",
+  "Europe & Central Asia": "Europe and Central Asia (WB)",
+  "East Asia & Pacific": "East Asia and Pacific (WB)",
+  "South Asia": "South Asia (WB)",
+  "Middle East & North Africa": "Middle East and North Africa (WB)",
+  "Sub-Saharan Africa": "Sub-Saharan Africa (WB)",
+  "European Union": "European Union",
+  "OECD members": "OECD members"
```

Comment:
Shouldn't this be `OECD members (WB)`, given that it's not an official OWID region?

Reply:
Same as in European Union. Unclear if the same entity definition propagates back in time.